app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):

```

Comment:
It is not a requirement to include ABC in this case. This class will have the same behavior with or without including it in parent class.